Python/optimizer.c

```diff
@@ -707,15 +706,13 @@ translate_bytecode_to_trace(
                                             PyUnicode_AsUTF8(new_code->co_qualname),
                                             PyUnicode_AsUTF8(new_code->co_filename),
                                             new_code->co_firstlineno);
-                                    ADD_TO_TRACE(SAVE_IP, 0, 0);
```

Comment:
Interesting. Because you added an assignment to `frame->prev_instr` to `_PUSH_FRAME`, you don't need this dummy `SAVE_IP` any more. Same when bailing on version mismatch below and ditto if `func == NULL`. Right?